arangod/Agency/Job.cpp

```diff
@@ -53,6 +53,8 @@ std::string const cleanedPrefix = "/Target/CleanedServers";
 std::string const toBeCleanedPrefix = "/Target/ToBeCleanedServers";
 std::string const failedServersPrefix = "/Target/FailedServers";
 std::string const planColPrefix = "/Plan/Collections/";
+std::string const planDBPrefix = "/Plan/Databases/";
```

Comment:
Finally constants.
I guess they are still in `AgencyStrings.h` in addition (just without the `/`)
I think we should at one point unify them and put all of them in a central Static Strings that is shared between ClusterInfo and Agency to avoid mess-up of paths.